### ISSUE TITLE
remove mgo session cloning

### ISSUE THREAD

**Issue by bodokaiser** [satisfeet/hoopoe-old#37]
At the moment we do a `c := session.Clone(); defer c.Close();` on each database action but on closer thoughts parallelism is not enforced per action but per request. This means that it makes more sense to clone a `mgo.Session` on each request then on each action.

Unfortunately this hard to implement at least if we want to avoid solutions like:

```
func (s *Store) Find(s *mgo.Session, models interface{}) error {
  // ...
}
```

One idea would be that we add some `.store()` helper to `httpd.handler` which will recreate a `store.Store` for us. This sounds a good solution but may bring downsides because recreation.

**Comment by bodokaiser** [comment]
There was a misconception on how connection pooling works on my side see [here](https://groups.google.com/forum/#!topic/mgo-users/uIAhDsAw4D4) for details.

The result is that there is no need to manually clone sessions as this is done by `mgo` itself. This issue should remove the unnecessary clone stuff.

**Comment by bodokaiser** [comment]
Fixed in #42.
